source/_docs/nested-docroot.md

```diff
@@ -43,7 +64,7 @@ Below we recommend using Git, but you can also use SFTP to set your site up for
 6. Follow the instructions in either [Create a New Site with a Nested Docroot](#create-a-new-site-with-a-nested-docroot) or [Convert an Existing Site to Use a Nested Docroot](#convert-an-existing-site-to-use-a-nested-docroot) below.
 
 ### Create a New Site with a Nested Docroot
-If your site utilizes a [Custom Upstream](/docs/custom-upstream/) with a `pantheon.yml` file that enables nested docroot and the CMS code is in a web subdirectory, you are good to go! Otherwise, create a new site and follow the steps below.
+If your site utilizes a [Custom Upstream](/docs/custom-upstream/) with a `pantheon.upstream.yml` file that enables nested docroot and the CMS code is in a web subdirectory, you are good to go! Otherwise, create a new site and follow the steps below.
```

Comment:
Alternately, we could leave this as-is for now, and come back and recommend using the Pantheon D7, D8 or WordPress upstream w/ relocated document root (once those exist).